docs/content/configuration/csi_ug_config_create_statefulset.md

```diff
@@ -1,127 +1,126 @@
 # Creating a StatefulSet
 
-Create a StatefulSet yaml file to manage stateful applications.
+Create a StatefulSet YAML file to manage stateful applications.
 
-The IBM® block storage CSI driver supports using both file system and raw block volume types.
+The IBM® block storage CSI driver supports both file system and raw block volume types.
 
 StatefulSets can include volumes with file systems, raw block volume systems, or both.
```

Comment:
I wouldn't refer to `raw block` as a "system"
```suggestion
StatefulSets can use raw block volumes, volumes with file systems, or both.
```

Reply:
in addition, a volume cannot have more than 1 file system.
would it be more accurate to change `volumes with file systems` to `volumes with a file system`/`file system volumes`?

Reply:
DONE